proto/storage/image.proto

```diff
@@ -57,8 +57,10 @@ message Image {
 }
 
 message DataSource {
-    string id   = 1;
-    string name = 2;
+    string id                = 1;
+    string name              = 2;
+    string mirror            = 3;
+    string mirror_cluster_id = 4;
```

Comment:
Since this just adds new fields to proto without any search tags or sql constraints, it should not need a migration.